java-api.md

```diff
@@ -199,6 +199,11 @@ Second generator would generate strings of length 8 from specified range of char
 "EF893232", "2E49D0AB", "BE129E15", "938FFC1C", "BB8A43ED", "829D1CA2", ...
 ```
 
+## Now methods
```

Comment:
Can we have now() which returns timestamp in millis? Thats more useful than these date functions.